src/pytorch_lightning/strategies/ddp_spawn.py

```diff
@@ -175,7 +174,6 @@ def set_world_ranks(self, process_idx: int = 0) -> None:
         rank_zero_only.rank = self.cluster_environment.global_rank()
 
     def _worker_setup(self, process_idx: int) -> None:
-        reset_seed()
```

Comment:
This is now handled in the launcher, so not necessary anymore. Would anyway not have been necessary when using fork.